tests/bpf2c_tests/expected/bindmonitor_dll.c

```diff
@@ -57,26 +57,26 @@ _get_hash(_Outptr_result_buffer_maybenull_(*size) const uint8_t** hash, _Out_ si
 static map_entry_t _maps[] = {
     {NULL,
      {
-         1,
-         8,
-         68,
-         1024,
-         0,
-         0,
-         0,
-         0,
+         1,    // Type of map.
```

Comment:
This should really be an enum symbol rather than an integer
```suggestion
         BPF_MAP_TYPE_HASH,
```